### DISCUSSION TITLE
Modding Buildings/Resources Guide

### DISCUSSION THREAD

**Discussion by PaxBotanica** [Keriew/augustus#986] [Q&A]
Is there a basic how-to on where to find building resources both code and graphics? and/or the process involved, I would love to help if I can.

**Comment by Keriew** [comment]
The project's [wiki](https://github.com/Keriew/augustus/wiki/Adding-a-new-building-checklist) has a list of steps needed to add a new building. I planned also adding one for adding new walkers but never got around to doing it. For graphics, take a look at the assets and their corresponding XML files.

**Reply by maoxp09** [reply]
I am about to complete a complete set of walking and death animation graphics for the "Walker", and I also want to contribute to it
